src/openforms/forms/migrations/0067_fix_typo_20230124_1624.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.16 on 2023-01-24 15:24
+
+from django.db import migrations
+
+from openforms.formio.utils import iter_components
+
+
+def fix_typo(apps, schema_editor):
+    FormDefinition = apps.get_model("forms", "FormDefinition")
+
+    form_definitions = FormDefinition.objects.all()
+
+    form_definitions_to_update = []
+    for form_definition in form_definitions:
+        updated_form_definition = False
+        for comp in iter_components(configuration=form_definition.configuration):
+            if comp["registration"]["attribute"] == "Strainitiator_straatat":
```

Comment:
this also needs a `FormVariable.objects.filter(prefill_attribute="...").update(prefill_attribute="...")` call, as a form variable should exist for every component.

See https://github.com/open-formulieren/open-forms/blob/master/src/openforms/forms/models/form_variable.py#L142

Reply:
What should I filter by and update to? The incorrect "Strainitiator_straatat" is not a `prefill_attribute` for any of the plugins. Perhaps I misunderstand the connection between `FormVariable`s and form components.